src/axom/primal/geometry/Polyhedron.hpp

```diff
@@ -661,9 +668,9 @@ class Polyhedron
  * \brief Creates a Polyhedron from a given Hexahedron's vertices.
  *
  * \param [in] hex The hexahedron
- * \param [in] checkSign If true (default is false), checks the volume of the
- *             Polyhedron is positive. If volume is negative, order of some
- *             vertices will be swapped.
+ * \param [in] checkSign If true (default is false), checks the signed volume
+ *             of the Polyhedron is positive. If signed volume is negative,
+ *             order of some vertices will be swapped.
```

Comment:
Minor:
```suggestion
 * \param [in] checkSign If true (default is false), checks if the signed volume
 *             of the Polyhedron is positive. If signed volume is negative,
 *             the order of some vertices will be swapped.
```

Reply:
Since many of these methods have similar comments ragarding signed volume being negative and reordering vertices, would it be better to add one overarching group comment about this and enclose all such methods within that documentation? That is, something like this: https://github.com/LLNL/axom/blob/develop/src/axom/sidre/core/Group.hpp#L143